packages/shorebird_cli/lib/src/commands/release/release_aar_command.dart

```diff
@@ -202,7 +202,7 @@ ${summary.join('\n')}
       release = await codePushClientWrapper.createRelease(
         appId: appId,
         version: releaseVersion,
-        flutterRevision: shorebirdEnv.flutterRevision,
+        flutterRevision: flutterRevision,
```

Comment:
```suggestion
        // Intentionally passing flutterRevision rather than shorebirdEnv.flutterRevision
        // since we've already reverted shorebirdEnv back to the latest flutter at this point.
        flutterRevision: flutterRevision,
```

Reply:
Updated the variable name to make it clear that we're passing the release flutter version